src/telemetry/traces.ts

```diff
@@ -17,6 +18,7 @@ export class Tracer {
     });
     this.tracer.register();
     this.applicationID = process.env.DBOS__APPID  || "APP_ID_NOT_DEFINED";
+    this.applicationVersion = process.env.DBOS__APPVERSION || "APP_VERSION_NOT_DEFINED";
```

Comment:
Instead of reading this from multiple places, should we add some `getAppVersion()` and `getAppID()` functions in config.ts?

Reply:
These are read/evaluated only once. I think actually using the environment is the easiest: no need for configuration, validation & co. If we added helpers, they would read the environment. If we have a use case for setting these using the configuration file (e.g., people would want to version these?) we could do something more sophisticated

Reply:
But you get different default values at different places. Sometimes it's an empty string while here is "APP_VERSION_NOT_DEFINED"... I think we may want to make it consistent.

Reply:
Ha, yes, so `APP_VERSION_NOT_DEFINED ` is really useful for the `Tracer`, not so much for the other parts. The logs will selectively print the version if it is specified. What's missing, I guess, is having `APP_VERSION_NOT_DEFINED` for the exported logs (not console).

@apoliakov do we feel that these placeholder values are useful? What about we set them to an empty string everywhere?

Reply:
So what does this actually do in the tracer?

Reply:
As we discussed in the meeting, we don't want to show app version or app ID if they're undefined in our traces.